### DISCUSSION TITLE
Does textgen need a Discord server?

### DISCUSSION THREAD

**Discussion by jtang613** [oobabooga/text-generation-webui#181] [Polls]
There's currently a lot of textgen discussion and support happening on Flexgen's Discord server. Would it help if this project had its own server?

**Comment by HappyCatKitten** [comment]
upvoted

**Comment by oobabooga** [comment]
The last thing I want is to turn this project into a discord circlejerk. I would prefer to keep open-ended discussions in the Discussions tab here on GitHub.

**Reply by olihough86** [reply]
preach!

**Reply by jtang613** [reply]
Fair enough. Was only thinking of a better place to offload the "How do I make it run?" questions, of which there seem to be many. But I'm happy to defer. Your project, your call.

**Reply by alexl83** [reply]
Would it be possible/make sense to link this github space with reddit r/oobabooga ? Se users could have a space for exchanges without cluttering github issues/PRs

**Comment by mcmonkey4eva** [comment]
ooba doesn't want a Discord, and that's okay, but I would really like a Discord for this - and since I'm quite experienced with running Discords, I figured I'd just go ahead and make an unofficial community Discord myself - you can join @ https://discord.gg/u27RhJECcW
